matminer/featurizers/composition.py

```diff
@@ -1320,3 +1319,123 @@ def citations(self):
+        self.mixing_data = {}
+        for a, b, dHf in mixing_dataset.itertuples(index=False):
+            key = tuple(sorted((a, b)))
+            self.mixing_data[key] = dHf
+
+        # Load in a table of elemental properties
+        self.elem_data = MagpieData()
+
+    def featurize(self, comp):
+        return [self.compute_omega(comp), self.compute_delta(comp)]
+
+    def compute_omega(self, comp):
+        """Compute Yang's mixing thermodynamics descriptor
+
+        :math:`\frac{T_m \Delta S_{mix}}{ |  \Delta H_{mix} | }`
+
+        Where :math:`T_m` is average melting temperature,
+        :math:`\Delta S_{mix}` is the ideal mixing entropy,
+        and :math:`\Delta H_{mix}` is the average mixing enthalpies
+        of all pairs of elements in the alloy
+
+        Args:
+            comp (Composition) - Composition to featurizer
+        Returns:
+            (float) Omega
+        """
+
+        # Get the element names and fractions
+        elements, fractions = zip(*comp.element_composition.\
+                                  fractional_composition.items())
+
+        # Get the mean melting temperature
+        mean_Tm = PropertyStats.mean(
+            self.elem_data.get_elemental_properties(elements, "MeltingT"),
+            fractions
+        )
+
+        # Get the mixing entropy
+        entropy = np.dot(fractions, np.log(fractions)) * 8.314 / 1000
+
+        # Get the mixing enthalpy
+        enthalpy = 0
+        for i, (e1, f1) in enumerate(zip(elements, fractions)):
+            for e2, f2 in zip(elements[:i], fractions):
+                key = tuple(sorted((e1.symbol, e2.symbol)))
+                enthalpy += f1 * f2 * self.mixing_data[key]
+        enthalpy *= 4
+
+        return abs(mean_Tm * entropy / enthalpy)
+
+    def compute_delta(self, comp):
+        """Compute Yang's delta parameter
+
+        :math:`\sqrt{\sum^n_{i=1} c_i \left( 1 - \frac{r_i}{\bar{r}} \right)^2 }`
+
+        where :math:`c_i` and :math:`r_i` are the fraction and radius of
+        element :math:`i`, and :math:`\bar{r}` is the fraction-weighted
+        average of the radii. We use the radii compiled by
+        .. Miracle et al. `https://www.tandfonline.com/doi/ref/10.1179/095066010X12646898728200?scroll=top`.
+
+        Args:
+            comp (Composition) - Composition to assess
+        Returns:
+            (float) delta
+
+        """
+
+        elements, fractions = zip(*comp.element_composition.items())
+
+        # Get the radii of elements
+        radii = self.elem_data.get_elemental_properties(elements,
+                                                        "MiracleRadius")
+        mean_r = PropertyStats.mean(radii, fractions)
+
+        # Compute the mean (1 - r/\bar{r})^2
+        r_dev = np.power(1.0 - np.divide(radii, mean_r), 2)
+        return np.sqrt(PropertyStats.mean(r_dev, fractions))
+
+    def feature_labels(self):
+        return ['Yang Omega', 'Yang delta']
```

Comment:
I would stick to a consistent capitalization unless there is good reason not to, e.g.

``Yang omega, Yang delta``